css/worksection.css

```diff
@@ -98,6 +96,69 @@
   color: #fff;
 }
 
+#modal-overlay {
+  position: fixed;
+  display: none;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0%;
+  opacity: 1;
+  background-color: #c1c7d0;
+  mix-blend-mode: multiply;
+  transition: 200ms ease-in-out;
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
+  pointer-events: all;
+}
+
+.modal-overlay .active {
+  opacity: 1;
+  pointer-events: all;
+}
+
+.modal {
+  position: fixed;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  display: none;
+  transition: 200ms ease-in-out;
+  z-index: 10;
+  border: 1px solid gray;
+  width: 600px;
+  max-width: 80%;
+  max-height: 80%;
+  overflow-y: scroll;
+  border-radius: 8px;
+  padding: 15px 5px 5px 20px;
+
+  /* margin: 15px; */
+  background: #fff;
+}
+
+/* .modal .active {
+  transform: translate(-50%, -50%) scale(1);
+} */
```

Comment:
- [ ] Commenting old block of code makes your code messy please try to avoid this instead use git to restore an old version of the code, so your code uses the best practices